selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Refactor.java

```diff
@@ -736,12 +657,32 @@ public int getQuantityLeasedLineInLeftEditor() {
    * @return quantity of leased lines
    */
   public int getQuantityLeasedLineInRightEditor() {
-    int lineQuantity =
-        new WebDriverWait(seleniumWebDriver, REDRAW_UI_ELEMENTS_TIMEOUT_SEC)
-            .until(
-                ExpectedConditions.presenceOfAllElementsLocatedBy(
-                    By.xpath(Locators.LEASED_LINE_RIGHT_EDITOR)))
-            .size();
-    return lineQuantity;
+    return redrawUiElementWait
+        .until(presenceOfAllElementsLocatedBy(By.xpath(LEASED_LINE_RIGHT_EDITOR)))
+        .size();
+  }
+
+  private void typeAndWaitTextInProvidedElement(WebElement element, String newValue) {
```

Comment:
`InProvidedElement` looks redundant. It could be explained in Java Docs above the method.